map_gui/src/render/road.rs

```diff
@@ -91,24 +91,22 @@ impl DrawRoad {
                     app.cs()
                         .zoomed_road_surface(LaneType::Driving, r.get_rank())
                 };
-
-                if false {
-                    // TODO Not ready yet
+                // TODO: find a good way to draw an appropriate background
+                if r.center_pts.quadrant() > 1 && r.center_pts.quadrant() < 4 {
+                    println!("quadrant: {}", r.center_pts.quadrant().to_string());
+                    println!("name: {}", r.get_name(None));
                     batch.append(Line(name).fg(center_line_color).render_curvey(
                         prerender,
-                        &r.center_pts,
+                        &r.center_pts.reversed(),
```

Comment:
Looks like the two branches only differ here. How about we move the conditional down here, something like `if r.center_pts.quadrant() > 1 && r.center_pts.quadrant() < 4 { &r.center_pts.reversed() } else { &r.center_pts }`. Conditionals are expressions, so they can be in the middle of an argument list

Reply:
The latest commit implements the requested changes, but I ran into issues with the borrow checker that I didn't understand how to resolve with this particular change request. When I used the expression as-is, rustc complains that we declare a temporary value which is freed while still in use. My hacky solution was to declare a variable before the call to ```batch.append()``` so that the return value of ```r.center_pts.reversed()``` is owned by another variable, but I'm sure a better solution exists.

Reply:
Ah, you're right -- this is an unfortunate limitation of the borrow checker. I would actually prefer the version you had previously with two calls to `render_curvey`, because then we don't calculate `center_pts.reversed()` everytime. I don't know if the compiler would be smart enough to optimize it away here, and this is a somewhat performance-sensitive bit.

I vote in this PR we leave the new curvey labels disabled until we're ready:

```
if false {
  if quadrant check {
    render_curvey reversed
   } else {
     render_curvey normal
   }
} else {
  the existing code that adds a background
}